ibm/mas_devops/roles/odh/tasks/determine-storage-classes.yml

```diff
@@ -0,0 +1,32 @@
+# Lookup storage class availabiity
+# -----------------------------------------------------------------------------
+- name: "Load default storage class information"
+  include_vars: "{{ role_path }}/../../common_vars/default_storage_classes.yml"
+
+- name: "determine-storage-classes : Lookup storage classes"
+  kubernetes.core.k8s_info:
+    api_version: storage.k8s.io/v1
+    kind: StorageClass
+  register: lookup_storageclasses
+
+- name: "Debug available storage classes"
+  debug:
+    msg: "{{ lookup_storageclasses | ibm.mas_devops.getResourceNames }}"
+
+# Set Primary Storage (Required)
+# -----------------------------------------------------------------------------
+- name: "determine-storage-classes : Default Primary Storage if not set by user"
+  when: cpd_primary_storage_class is not defined or cpd_primary_storage_class == ""
+  set_fact:
+    cpd_primary_storage_class: "{{ lookup_storageclasses | ibm.mas_devops.defaultStorageClass(default_storage_classes_rwx) }}"
+
+- name: "determine-storage-classes : Assert that primary storage class has been defined"
+  assert:
+    that: cpd_primary_storage_class is defined and cpd_primary_storage_class != ""
```

Comment:
we are using this for define storage classes because then we use this specific class in our deployment for example: https://github.com/ibm-mas/ansible-devops/blob/04e507bd313ab7f35cbb74e5eb3bc7112217ff63/ibm/mas_devops/roles/odh/templates/odh/dspa-aws.yml.j2#L37

Reply:
this is fixed now